modules/tests/jvm/src/main/scala/ffstest/FFrameworkPlatform.scala

```diff
@@ -4,31 +4,6 @@
 
 package ffstest
 
-import cats.effect.{IO, Resource}
 import munit.CatsEffectSuite
-import io.opentelemetry.api.GlobalOpenTelemetry
-import org.typelevel.otel4s.java.OtelJava
-import org.typelevel.otel4s.trace.Tracer
 
-trait FTestPlatform extends CatsEffectSuite {
-  def tracerResource: Resource[IO, Tracer[IO]] = 
-    Resource.eval(IO(GlobalOpenTelemetry.get()))
-      .evalMap(OtelJava.forAsync(_))
-      .evalMap(_.tracerProvider.get(getClass.getName()))
-
-  private var ioTracerFinalizer: IO[Unit] = _
-
-  implicit lazy val ioTracer: Tracer[IO] = {
-    val tracerAndFinalizer = tracerResource.allocated.unsafeRunSync()
```

Comment:
We cannot use `unsafeRunSync()` on Native and Scala.js platforms. So the tests will be slightly more verbose.